spec.bs

```diff
@@ -360,6 +401,8 @@ to no longer be entitled to the purchase after it is consumed.
     <a href="https://en.wikipedia.org/wiki/ISO_8601#Durations">ISO 8601 duration</a>,
     in which the item costs the {{ItemDetails/introductoryPrice}}. After this
     period the item is epected to cost the {{ItemDetails/price}}.
+  * {{ItemDetails/introductoryPriceCycles}} is the number of subscription cycles
+    during which the {{ItemDetails/introductoryPrice}} is effective.
 </div>
 
 ## PurchaseDetails dictionary ## {#purchaseDetails-dictionary}
```

Comment:
In this section:
"A {{PurchaseDetails}} dictionary represents information about a digital item
  from a {{Window/getDigitalGoodsService(serviceProvider)/serviceProvider}} to
  which the user is currently entitled"

Replace "to which the user is currently entitled".with something like "which the user has purchased at some point" (feel free to use some better wording here)